app/assets/stylesheets/ama_layout/webfonts/ss-symbolicons-block.scss

```diff
@@ -0,0 +1,50 @@
+@charset "UTF-8";
```

Comment:
after looking at other gems it doesn't look like they do this - so my previous comment can be ignored.